laravel/the-basics/databases.html.md

```diff
@@ -249,6 +249,13 @@ SELECT datname FROM pg_database;
 (4 rows)
 ```
 
+<aside class="callout">
+There are tons of ways to interact with your Postgres Fly App! If you want to get a bird's eye view on your Postgres database list outside of the `flyctl postgres connect` command, simply run the command below instead.
+</aside>
+```cmd 
+flyctl postgres db list -a <postgres-fly-app-name>
```

Comment:
@rugwirobaker, here's that **flyctl postgres db lis** command!
I've added sqlite integration below as well, check it out as well!